src/__tests__/proposals.int.test.ts

```diff
@@ -60,13 +84,37 @@ describe('/proposals', () => {
                   applicantId: 1,
                   opportunityId: 1,
                   createdAt: expect.stringMatching(isoTimestampPattern) as string,
+                  versions: [],
                 },
                 {
                   id: 1,
                   externalId: 'proposal-1',
                   applicantId: 1,
                   opportunityId: 1,
                   createdAt: expect.stringMatching(isoTimestampPattern) as string,
+                  versions: [{
```

Comment:
Note that this structure / this test is identical to the test in #313 -- which is to say, both of these implementations result in the same API result.